src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -302,6 +302,14 @@ export default {
 			}
 		},
 	},
+
+	mounted() {
+		const titleSpan = document.getElementById(`conversation_${this.item.token}`)?.querySelector('.line-one__title')
+		if (!titleSpan) return
```

Comment:
```suggestion
		if (!titleSpan) {
			return
		}
```

Reply:
Got it, will fix tommorow. But if it's accepted by team, maybe it should be regulated by linter? 
https://eslint.org/docs/latest/rules/curly